glom/core.py

```diff
@@ -1364,6 +1403,34 @@ def _t_eval(target, _t, scope):
             # if args to the call "reset" their path
             # e.g. "T.a" should mean the same thing
             # in both of these specs: T.a and T.b(T.a)
+        else:  # arithmetic operators
+            try:
+                if op == '+':
+                    cur = cur + arg
+                elif op == '-':
+                    cur = cur - arg
+                elif op == '*':
+                    cur = cur * arg
+                #elif op == '#':
+                #    cur = cur // arg  # TODO: python 2 friendly approach?
```

Comment:
do we need floor division?

Reply:
Or wait, if both the user and we imported future division, wouldn't this work fine?

Reply:
oh doh forgot about that